packages/edit-site/src/components/editor/index.js

```diff
@@ -201,6 +205,12 @@ function Editor( { initialSettings } ) {
 										settings.__experimentalGlobalStylesBaseStyles
 									}
 								>
+									{
+										// Template previews need the editor styles to be available.
+										<EditorStyles
+											styles={ settings.styles }
+										/>
```

Comment:
Is this needed because the editor styles are only rendered inside the iframe while the previews are outside the iframe? I think @ellatrix has an alternative fix for this (using iframe for previews somewhere)

Reply:
Yes, #28165 would be the preferred fix.